spec/features/toc_links_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Table of Contents links", type: :feature, js: true do
+  describe "links in the table of contents" do
+    it "have real hrefs, not just '#'" do
+      # Visit a collection page that has a table of contents
+      visit "/catalog/AC198"
```

Comment:
This particular fixture does not have subcomponents to render in a ToC. Changing the fixture to MC221 makes the test pass.